delegation-toolkit/concepts/advanced-permissions.md

```diff
@@ -23,42 +23,43 @@ Dapps can use this method to request a wallet to grant the dapp permission to ex
 `wallet_grantPermissions` requires a `signer` parameter, which identifies the entity requesting or managing the permission.
 Common signer implementations include wallet signers, single key and multisig signers, and account signers.
 
-The Smart Accounts Kit supports multiple types of signers, but [an account signer is used in this documentation's examples](../guides/erc7715/execute-on-metamask-users-behalf.md) as a common implementation.
-With an account signer, a session account is created and used for the single purpose of requesting and redeeming ERC-7715 permissions, and does not contain tokens.
+The Smart Accounts Kit supports multiple types of signers, but [an account signer is used in this documentation's examples](../guides/advanced-permissions/execute-on-metamask-users-behalf.md) as a common implementation.
+With an account signer, a session account is created and used for the single purpose of requesting and redeeming
+Advanced Permissions, and does not contain tokens.
```

Comment:
```suggestion
When you use an account signer, a session account is created solely to request and redeem Advanced Permissions, and doesn't contain tokens
```

Reply:
I know this is already merged, but missing a period here.

Reply:
Nice catch, have you included in any other PR already? If not, I can push with the PR I'm working on.

Reply:
Feel free to add to your PR -- I will work on another PR updating all the URLs/redirects on Monday and address any remaining fixes then.